packages/terra-form-select/docs/MigrationGuide.md

```diff
@@ -0,0 +1,87 @@
+# Migrating from terra-form
```

Comment:
Were we including the Migration Guide in the docs section or release notes?

(just realized I need to write this too...)